src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/build/AdditionalFiles/vs-threading.MembersRequiringMainThread.txt

```diff
@@ -80,12 +108,14 @@
 ![Microsoft.VisualStudio.Shell.Interop.IVsRunningDocumentTable2]
 ![Microsoft.VisualStudio.Shell.Interop.IVsRunningDocumentTable3]
 ![Microsoft.VisualStudio.Shell.Interop.IVsRunningDocumentTable4]
-![Microsoft.VisualStudio.Shell.Interop.IVsRunningDocumentTable5]
```

Comment:
Deliberate, neither of these two removes are free-threaded. Both require the UI thread to call their methods.